README.md

```diff
@@ -1,5 +1,5 @@
 <p align=center>
-    <img alt="Cloak Logo" src="app/static/img/logo.png" width="200px" height="200px">
+    <img alt="Cloak Logo" src="app/static/img/newlogo.png" width="200px" height="200px">
```

Comment:
We should just replace the logo instead of creating a new file. The old one will always be available in the git history.

Reply:
I had a weird bug where I replaced the logo and kept the same name, yet it was still loading the old logo.

Reply:
I'll fix it